pkg/statistics/handle/ddl/ddl.go

```diff
@@ -41,73 +40,24 @@ func NewDDLHandler(
 	globalStatsHandler types.StatsGlobal,
 ) types.DDL {
 	return &ddlHandlerImpl{
-		ddlEventCh:         make(chan *util.DDLEvent, 1000),
+		ddlEventCh:         make(chan *ddlutil.SchemaChangeEvent, 1000),
 		statsWriter:        statsWriter,
 		statsHandler:       statsHandler,
 		globalStatsHandler: globalStatsHandler,
 	}
 }
 
 // HandleDDLEvent begins to process a ddl task.
-func (h *ddlHandlerImpl) HandleDDLEvent(t *util.DDLEvent) error {
+func (h *ddlHandlerImpl) HandleDDLEvent(s *ddlutil.SchemaChangeEvent) error {
 	sctx, err := h.statsHandler.SPool().Get()
 	if err != nil {
 		return err
 	}
 	defer h.statsHandler.SPool().Put(sctx)
 
-	// ActionFlashbackCluster will not create any new stats info
-	// and it's SchemaID alwayws equals to 0, so skip check it.
-	if t.GetType() != model.ActionFlashbackCluster && t.SchemaChangeEvent == nil {
-		if isSysDB, err := t.IsMemOrSysDB(sctx.(sessionctx.Context)); err != nil {
-			return err
-		} else if isSysDB {
-			// EXCHANGE PARTITION EVENT NOTES:
```

Comment:
Better to keep this comment somewhere. We allow users to exchange one normal table for a system table.